src/Sulu/Component/Content/Metadata/Loader/StructureXmlLoader.php

```diff
@@ -62,6 +62,8 @@ class StructureXmlLoader extends AbstractLoader
         'navContexts',
         'shadow-on',
         'shadow-base',
+        'isUpdated',
+        'updated',
```

Comment:
The website/frontend should not get the is / enabled flag it just for the admin. So if possible we don't write it here.

```suggestion
        'lastModified',
```